core/controllers/base.py

```diff
@@ -398,6 +398,11 @@ def render_template(
 
         self.response.expires = 'Mon, 01 Jan 1990 00:00:00 GMT'
         self.response.pragma = 'no-cache'
+
+        # cache_slug passed to the template should not have trailing or ending
```

Comment:
It's not useful to add this here; I suggest putting it where the cache slug is defined. Otherwise no one's going to discover it...

Also, it's always better to add a test. This condition should be easy to check; could you add a test that does so?